### PR TITLE
Adds golang server example

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
r? @adreyfus-stripe @ctrudeau-stripe @tomer-stripe

### CODE REVIEW

File: server/go/server.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  }
+
+  fmt.Println(os.Getenv("STRIPE_SECRET_KEY"))
+
+  stripe.Key = os.Getenv("STRIPE_SECRET_KEY")
+  e := echo.New()
+  e.Use(middleware.Logger())
+  e.Use(middleware.Recover())
+  e.Renderer = echoview.Default()
+  e.Static("/", "../../client")
```
Comment (adreyfus-stripe) [inline server/go/server.go]: Can we use STATIC_DIR set in the env? So that downloading with the CLI won't break it

**Comment by adreyfus-stripe** [comment]
lgtm after that one comment. thank you!

File: server/go/server.go (hunk 1 above)
Comment (cjavilla-stripe) [reply server/go/server.go]: good call. updated

**Comment by cjavilla-stripe** [comment]
Thanks for looking, @adreyfus-stripe! updated that one reference.

**Pull request merged**